src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -47,6 +47,13 @@ def add_suggestion(kw, suggestion):
                 logger.info("Deprecated keyword %s", kw)
                 suggestions.append(suggestion)
 
+        if content.hasKey("RUNPATH") and "%d" in content.getValue("RUNPATH"):
+            add_suggestion(
+                "RUNPATH",
+                "The use of %d in RUNPATH has been deprecated. "
+                "Instead use equivalent RUNPATH syntax: realization-<IENS>/iter-<ITER>",
```

Comment:
Is "syntax" too technical here? Should we try something simpler like "Instead use the <IENS> <ITER> keywords, e.g.:  realization-<IENS>/iter-<ITER>"

Reply:
Np. :+1: